### PR TITLE
Django 1.9 support

### PR DESCRIPTION
**Pull request by kaedroho**

### CODE REVIEW

**Comment by kaedroho** [comment]
Ready for review!

File: tox.ini [hunk 1]
```diff
@@ -17,6 +18,7 @@ deps =
     six>=1.6.1
     dj17: Django>=1.7,<1.8
     dj18: Django>=1.8,<1.9
+    dj19: Django==1.9,<1.10
```
Comment (zerolab) [inline tox.ini]: Should this not be `Django>=1.9,<1.10` ?

Comment (kaedroho) [inline tox.ini]: Oops, cheers!

**Comment by gasman** [comment]
Other than the above, looks good to me!

**Comment by kaedroho** [comment]
Thanks for the review! Fixes made. Happy for me to merge this?

**Comment by gasman** [comment]
Merged in a007083cb64c52c1a3dcfa762e5c213863d087bb + parents.

**Pull request closed without merging**